Moving the issue_template.md to the docs folder to increase code cov

thomad74 opened the pull request:
# Description of Pull Request

Moving the file for the issue template to the correct folder (docs) to try and fix problems regarding code coverage. Also renamed the issueTemplate.md to issue_template.md to match the conventions of the pull_request_template.md file in the same folder.

Fixes #183 


## Type of Change

Please describe the pull request as one of the following: 

- [x] Bug fix
- [ ] Breaking change
- [ ] New feature
- [x] Documentation update
- [ ] Other (Please specify any helpful information below)

## Tags

@Lancasterwu

thomad74 commented:
Should this be in the hidden `.git` folder? I am not seeing a `.github` folder anywhere in the `master` branch.

Michionlion commented:
> Should this be in the hidden `.git` folder? I am not seeing a `.github` folder anywhere in the `master` branch.

No, the `.git` folder is not put on github. If the `.github` folder does not exist, you can create it. It is likely that the PR template should also be moved, and the Logo PR #155 also needs to be changed since the `docs` folder will be unused.

thomad74 commented:
Okay, I just created a `.github` folder and moved the `issue_template.md` and the `pull_request_template.md` file into it. I also deleted the `docs` folder as @Lancasterwu said it will be unused now.
